scripts/test_examples.sh

```diff
@@ -3,34 +3,20 @@
 TMP_FILE="/tmp/out"
 exitCode=0
 PREV_IFS="$IFS"
+RUNTIME_LIMIT=5
 
-# test whether output of the command contains expected lines
+# tests whether the output of the command contains expected lines
 # arguments
-# 1-st command to run
-# 2-nd array of expected strings in the command output
-# 3-rd argument is an optional array of unexpected strings in the command output
-# 4-th argument is an optional command runtime limit
-function testOutput {
-IFS="${PREV_IFS}"
-
-    #run the command
-    if [ $# -ge 4 ]; then
-        $1 > ${TMP_FILE} 2>&1 &
-        CMD_PID=$!
-        sleep $4
-        kill $CMD_PID
-    else
-        $1 > ${TMP_FILE} 2>&1
-    fi
-
+# 1st command to run
+# 2nd array of expected strings in the command output
+# 3rd argument is an optional array of unexpected strings in the command output
+function testExpectedMessage {
```

Comment:
These changes are already part of ODPM-593 pull request.
For each newly started task create a separate branch on top of master or pantheon-dev (not on a top of another branch currently in review). This way you will avoid putting your code into multiple pull requests.
Let's finalize ODPM-593 pull request first, then we will merge master into this one and fix conflicts.